src/views/auth.py

```diff
@@ -52,32 +55,3 @@ def register():
 @login_required
 def account():
     return render_template('account.html', name=current_user.email)
-
-
-"""
-@app.route('/login', methods=['GET', 'POST'])
-def login():
-    form = LoginForm()
-    if request.method == 'POST' and form.validate():
-        print(request.form['email'])
-        existing_user = User.get_by_email(request.form['email'])
-        if existing_user:
-            print("yatta desu ne")
-            login_user(existing_user)
-            return redirect(url_for('account'))
-    return render_template('login.html', form=form)
-
-
-@app.route('/register')
-def register():
-    form = RegisterForm()
-    if request.method == 'POST' and form.validate():
-        existing_user = User.objects(email=form.email.data).first()
-        if existing_user is None:
-            hashpass = generate_password_hash(form.password.data, method='sha256')
-            hey = User(form.name.data, form.email.data, hashpass).save()
-            login_user(hey)
-            return redirect(url_for('account'))
-    
-    return render_template('register.html', form=form)
-"""
```

Comment:
Could you give me a bit more information about what is happening here?
Seems like the function exists multiple times in this script and that's why you removed it.
Wasn't there a conflict previously? I am a bit puzzled by this.

Reply:
this was already commented out (using """ and """) by the previous owner of the code. I think it was because of test purposes earlier in development. I simply did not need it, so I removed it.